service_storagefacility/app/services/storage/NodeService.scala

```diff
@@ -45,7 +44,10 @@ trait NodeService {
       mid: MuseumId,
       nodeId: StorageNodeDatabaseId,
       envReq: EnvironmentRequirement
-  )(implicit currUsr: AuthenticatedUser): Future[Option[EnvironmentRequirement]] = {
+  )(
+      implicit currUsr: AuthenticatedUser,
+      ec: ExecutionContext
```

Comment:
It's good practice, really. It's a trait that exposes reusable functionality. And it can be used in a context where you want to pass in a different execution context. E.g. when we have the need for an Actor to pull data from the datastore to index, some of these extension services can be useful. But we definitely don't want to use the default play context then. Instead we want to use the actors dispatcher context.
In fact...I'd argue that all our DAO's should have an implicit exec context passed implicitly instead of them importing the default one from play. But that's a big effort in itself.
For this PR it made sense since the code is "new".